src/nasir/A9_Shop.java

```diff
@@ -0,0 +1,57 @@
+package nasir;
+class A9_Shop {
+	int availableMaggie = 50;
+	int availableDosa =43;
+	int availablePouches =39;
+	int availablePanipuri=43;
+	int availableMasala=73;
+	
+	void purchaseItem(int maggie1, int dosa1, int pouches1,int panipuri1, int masala1){
+		if(availableMaggie>=maggie1 && availableDosa>=dosa1 && availablePouches>=pouches1 && availablePanipuri>=panipuri1 && availableMasala>=masala1){
```

Comment:
Take a look of below code
https://github.com/KrishnaKTechnocredits/JAVATechnoDec2021/pull/209/files